DANS-KNAW/easy-stage-dataset
# EASY-2358: message for datamanager from EMD to AMD

fixes EASY-2358: message for datamanager from EMD to AMD

#### When applied it will
* 
* 
* 

#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?

* a fresh `deasy-2019-10-02.box`
* [x] sword2 (and split-multi-depostit) should not be affected so try both before and after deploy.

      cd ~/git/service/easy/easy-sword2-dans-examples
      ./run.sh Simple https://deasy.dans.knaw.nl/sword2/collection/1 IPDBSTest IPDBSTest src/main/resources/agreement-flow/valid/audiences
* [x] build this, then build and deploy `easy-ingest-flow` + https://github.com/DANS-KNAW/easy-deposit-api/pull/181
* repeat the sword2 step, download emd/amd of both datasets, they should be identical except for timestamps and identifiers

      http://deasy.dans.knaw.nl:8080/fedora/objects/easy-dataset:18/datastreams
* submit a deposit with the deposit-ui
* [x] webui should not throw an exception
```
Unable to load model object:
nl.knaw.dans.common.lang.service.exceptions.ServiceException: nl.knaw.dans.common.lang.repo.exception.ObjectDeserializationException: nl.knaw.dans.common.lang.xml.XMLDeserializationException: org.jibx.runtime.JiBXException: No unmarshaller for element "{https://easy.dans.knaw.nl/easy/dataset-administrative-metadata/}administrative-md"
```

#### related pull requests on github
repo                       | PR
-------------------------- | -----------------
easy-                      | [PR#](PRlink)
